source/includes/documentation/_vue_js_elements.md

```diff
@@ -2280,36 +2280,73 @@ __Vuetify v2.6.14__ code example:
 
 #### 5.27.1 Progress circular
 
-[Vuetify documentation page](https://vuetifyjs.com/en/components/progress-circular/)
-
-- __Java__: _package com.epam.jdi.light.vuetify.elements.common.ProgressSpinner.java_
-
 ```java
-    @Test(dataProvider = "sizeWidthProgressSpinnersTestsDataProvider",
-      dataProviderClass = ProgressSpinnersDataProvider.class)
-    public void sizeWidthProgressCircularsTests(int index, String color, String height, String width) {
-      sizeWidthProgressSpinners.get(index).is().displayed();
-      sizeWidthProgressSpinners.get(index).is().spinning();
-      sizeWidthProgressSpinners.get(index).has().color(color);
-      sizeWidthProgressSpinners.get(index).has().height(height);
-      sizeWidthProgressSpinners.get(index).has().width(width);
-    }
+//@FindBy(css = "#ColorProgress .v-progress-circular" 
+@UI("#ColorProgress .v-progress-circular")
+public static List<ProgressCircular> colorProgressCirculars;
+
+@Test(dataProvider = "colorProgressCircularsTestsDataProvider",
+dataProviderClass = ProgressCircularDataProvider.class)
+public void colorProgressCircularsTests(int index, String color, String height, String width, int value) {
```

Comment:
Could you, please, create task for refactoring: add description to all the tests.
Thanks in advance

Reply:
Done. Do I need to add some description? It looks not quite clear)
https://github.com/jdi-testing/jdi-light/issues/4749